Assets/Scripts/PlayerInteraction/AnimalMovement.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Anivision.Core;
+using Anivision.PlayerInteraction;
+using Anivision.Vision;
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEditorInternal;
+#endif
+using UnityEngine;
+
+namespace Anivision.PlayerInteraction
+{
+    /// <summary>
+    /// Script that contains the movement information for an animal
+    /// Developers/Artists can check various parameters through the inspector
+    /// </summary>
+    public class AnimalMovement : MonoBehaviour
+    {
+        public bool canFly;
```

Comment:
If you've set a default value for canTeleport, shouldn't you also set a default value for canFly?

Reply:
Default value for booleans in C# is false! but can make this more explicit

Reply:
Nevermind then, I'm used to less strictly typed languages

Reply:
I think adding it would be helpful for future peeps